lang/en.js

```diff
@@ -189,6 +180,30 @@ export default {
   "reset": "Reset",
   "loadingResults": "Loading results",
   "edmIsShownAtLinkAlt": "View at the provider's website",
+  "record": {
+    "extendedInformation": "Extended information",
+    "hideAll": "Hide all information",
+    "record": "Record",
+    "showAll": "Show all information",
+    "similarItems": "Similar items",
+    "view": {
+      "pdf": "View PDF"
+    }
+  },
+  "contentfulManual": {
```

Comment:
What is this for?

Reply:
These were all strings that are not in the project, but Ash wanted to have them translated for the demo. So he got the translations done in lokalise and then copied them to contentful manually.